hazelcast/near_cache.py

```diff
@@ -171,6 +172,12 @@ def __getitem__(self, key):
             else value_record.value
         )
 
+    def _get_size_of_near_cache(self):
+        if "PyPy" in sys.version:
+            return 0
+        else:
+            return getsizeof(self)
+
```

Comment:
`getsizeof` can take an argument for the default, I think we can just use it instead of checking for the PyPy version:
```
"owned_entry_memory_cost": getsizeof(self, 0),
```

Reply:
Fixed at https://github.com/hazelcast/hazelcast-python-client/pull/571/commits/3f20f47c2540ebb4248f33ffd8deb8e977962a55.